.github/workflows/nightly-tests.yml

```diff
@@ -0,0 +1,77 @@
+# This workflow runs ECM's nightly tests, which currently only includes performance tests
+
+name: Nightly Tests
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 7 * * *' # run at 7 AM UTC, 2 AM ET.
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  performance-tests:
+    runs-on: [self-hosted]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get version files from terra-helmfile
+        env: 
+          BROADBOT_TOKEN: ${{ secrets.BROADBOT_TOKEN }}
+        run: |
+          curl -H 'Authorization: token $BROADBOT_TOKEN' \
+            -H 'Accept: application/vnd.github.v3.raw' \
+            -L https://api.github.com/repos/broadinstitute/terra-helmfile/contents/versions/app/dev.yaml \
+            --create-dirs -o "integration/terra-helmfile/versions/app/dev.yaml"
+          curl -H 'Authorization: token $BROADBOT_TOKEN' \
+            -H 'Accept: application/vnd.github.v3.raw' \
+            -L https://api.github.com/repos/broadinstitute/terra-helmfile/contents/environments/live/perf.yaml \
+            --create-dirs -o "integration/terra-helmfile/environments/live/perf.yaml"
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+
+      - name: Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ github.ref }}-${{ github.sha }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ github.ref }}
+
+      # Github workflows shouldn't have access to vault, so we read from GitHub secrets instead.
+      - name: Render GitHub Secrets
+        env:
+          DEV_FIRECLOUD_ACCOUNT_B64: ${{ secrets.DEV_FIRECLOUD_ACCOUNT_B64 }}
+          PERF_TESTRUNNER_ACCOUNT_B64: ${{ secrets.PERF_TESTRUNNER_ACCOUNT_B64 }}
+        run: |
+          echo "$DEV_FIRECLOUD_ACCOUNT_B64" | base64 -d >"./integration/src/main/resources/rendered/user-delegated-sa.json"
+          echo "$PERF_TESTRUNNER_ACCOUNT_B64" | base64 -d >"./integration/src/main/resources/rendered/testrunner-sa.json"
+
+      - name: Run the perf test suite
+        run: ./gradlew --build-cache :integration:runTest --args="suites/FullPerf.json /tmp/test-results"
+
+      - name: Upload results to Google Bucket
+        run: ./gradlew :integration:uploadResults --args="CompressDirectoryToTerraKernelK8S.json /tmp/test-results"
```

Comment:
Try replace `:integration:runTest` by `runTest` and `:integration:uploadResults` by `uploadResults` and rerun the test.